### ISSUE TITLE
Toast message when opened the app.

### ISSUE THREAD

**Issue by asaikarthikeya** [enviroCar/enviroCar-app#829]
Currently, When opened for the app for the first time, welcome message is shown to the user if he is loged in. During this message user cannot perform any action with the bottom navigation bar.

This is not a good feature in high end mobile phone because, the intial splash screen is shown 2 sec and toast is shown 2 sec, So user has to wait 4 sec(approx) to perform an action.

Possible solution - Remove the toast message when opened the app, show toast only after login and if possible reduce the splash screen time to 1sec.

**Comment by asaikarthikeya** [comment]
@SebaDro Please have a look at this issue. related issue -#366

**Comment by SebaDro** [comment]
Implemented by #835

**Comment by asaikarthikeya** [comment]
Merged into the master branch!
